agent/app/agent_unix.go

```diff
@@ -192,8 +192,10 @@ func contains(capabilities []string, capability string) bool {
 // object
 func (agent *ecsAgent) initializeResourceFields() {
 	agent.resourceFields = &taskresource.ResourceFields{
-		Control: cgroup.New(),
-		IOUtil:  ioutilwrapper.NewIOUtil(),
+		Control:      cgroup.New(),
```

Comment:
Is it a concern that we initialize these fields irrespective of whether resources like cgroup/volumes are enabled or not?

Reply:
It's not a big concern to me, as it's just creating the object. This avoids an additional check, if you feel strongly about this, I can change it.